src/Skeleton/PostCreateProject.php

```diff
@@ -0,0 +1,310 @@
+            'sass-loader@^7.0.1',
+        ];
+        if ($io->isVerbose()) {
+            $io->write(
+                sprintf(
+                    '   Install packages (%1$s) that are required for our FrameworkStylePackage.',
+                    implode(', ', $packages)
+                )
+            );
+        }
+        $output = shell_exec(sprintf('npm install %1$s --save-dev', implode(' ', $packages)));
+        if ($io->isVerbose()) {
+            $io->write($output);
+        }
+
+
+        $io->notice('→ Import our Framework JS');
+        if ($io->isVerbose()) {
+            $io->write('   Import the frameworkstylepackage index file');
+        }
+        $content = file_get_contents($projectDir . '/assets/js/app.js');
+        $insert = [
+            'import { Framework } from \'frameworkstylepackage/src/js/Index\'',
+        ];
+        $matches = [];
+        preg_match('|import\s.*css\/app\.(s)?css|', $content, $matches, PREG_OFFSET_CAPTURE);
+        $offset = mb_strpos($content, "\n", $matches[0][1]);
+        $content = self::insertStringAtPosition(
+            $content,
+            $offset,
+            "\n" . implode("\n", $insert)
+        );
+
+
+        $io->notice('→ Initialize Framework JS');
+        if ($io->isVerbose()) {
+            $io->write('   Create new instance of the Framework object');
+        }
+        $insert = [
+            'new Framework()',
+        ];
+        $content = self::insertStringAtPosition(
+            $content,
+            mb_strlen($content),
+            "\n" . implode("\n", $insert) . "\n"
+        );
+
+
+        $io->notice('→ Replace app.css with app.scss');
+        $content = preg_replace('|app\.css|', 'app.scss', $content);
+
+
+        // store the file
+        file_put_contents($projectDir . '/assets/js/app.js', $content);
+
+        // fix code styling, as the default
+        if ($io->isVerbose()) {
+            $io->write(
+                '   Apply StandardJS as the default app.js is not following these standards.'
+            );
+        }
+        shell_exec(' node_modules/.bin/standard assets/js/app.js --quiet --fix');
+
+
+        if (file_exists($projectDir . '/assets/css/app.css') && !file_exists($projectDir . '/assets/css/app.scss')) {
+            $io->notice('→ Rename app.css to app.scss');
+            if ($io->isVerbose()) {
+                $io->write(
+                    '   We use SCSS instead of CSS files, so we rename the original file.'
+                );
+            }
+            rename($projectDir . '/assets/css/app.css', $projectDir . '/assets/css/app.scss');
+        }
+
+
+        $io->notice('→ Import sccs-files');
+        if ($io->isVerbose()) {
+            $io->write(
+                '   Import bootstrap variables and our base scss-file.'
+            );
+        }
+        $content = file_get_contents($projectDir . '/assets/css/app.scss');
+        $insert = [
+            '@import \'~bootstrap/scss/functions\';',
```

Comment:
De eerste twee imports hier zijn overbodig denk ik. Dit is de file die we met de derde import binnenhalen: https://github.com/sumocoders/FrameworkStylePackage/blob/master/src/sass/style.scss. Daarin importen we al de functions en variables van bootstrap, dus deze hier zijn redundant.

Zie https://github.com/sumocoders/application-skeleton/pull/6